src/screens/positions/LpPositionListItem.tsx

```diff
@@ -137,6 +137,7 @@ export const LpPositionListItem: React.FC<Props> = ({ assets, totalAssetsValue,
                   </Box>
                   <LpRangeBadge
                     assets={assets.map((underlying, index) => ({
+                      id: underlying.asset.asset_code,
```

Comment:
possible for two LPs to have the same asset? If so this would create double indices

Reply:
These are for each LP pool, and to my knowledge there are no LP pools that allow you to pool two of the same asset, I don't think that would make sense.